niryo_robot_msgs/RobotState pose
---
int32 status
string message
float32[] joints
